python/paddle/fluid/incubate/fleet/tests/fleet_deep_ctr.py

```diff
@@ -0,0 +1,204 @@
+    )
+    """ network definition """
+    dnn_data = fluid.layers.data(
+        name="dnn_data",
+        shape=[-1, 1],
+        dtype="int64",
+        lod_level=1,
+        append_batch_size=False)
+    lr_data = fluid.layers.data(
+        name="lr_data",
+        shape=[-1, 1],
+        dtype="int64",
+        lod_level=1,
+        append_batch_size=False)
+    label = fluid.layers.data(
+        name="click",
+        shape=[-1, 1],
+        dtype="int64",
+        lod_level=0,
+        append_batch_size=False)
+
+    datas = [dnn_data, lr_data, label]
+
+    # build dnn model
+    dnn_layer_dims = [128, 64, 32, 1]
+    dnn_embedding = fluid.layers.embedding(
+        is_distributed=False,
+        input=dnn_data,
+        size=[dnn_input_dim, dnn_layer_dims[0]],
+        param_attr=fluid.ParamAttr(
+            name="deep_embedding",
+            initializer=fluid.initializer.Constant(value=0.01)),
+        is_sparse=True)
+    dnn_pool = fluid.layers.sequence_pool(input=dnn_embedding, pool_type="sum")
+    dnn_out = dnn_pool
+    for i, dim in enumerate(dnn_layer_dims[1:]):
+        fc = fluid.layers.fc(
+            input=dnn_out,
+            size=dim,
+            act="relu",
+            param_attr=fluid.ParamAttr(
+                initializer=fluid.initializer.Constant(value=0.01)),
+            name='dnn-fc-%d' % i)
+        dnn_out = fc
+
+    # build lr model
+    lr_embbding = fluid.layers.embedding(
+        is_distributed=False,
+        input=lr_data,
+        size=[lr_input_dim, 1],
+        param_attr=fluid.ParamAttr(
+            name="wide_embedding",
+            initializer=fluid.initializer.Constant(value=0.01)),
+        is_sparse=True)
+    lr_pool = fluid.layers.sequence_pool(input=lr_embbding, pool_type="sum")
+
+    merge_layer = fluid.layers.concat(input=[dnn_out, lr_pool], axis=1)
+
+    predict = fluid.layers.fc(input=merge_layer, size=2, act='softmax')
+    acc = fluid.layers.accuracy(input=predict, label=label)
+    auc_var, batch_auc_var, auc_states = fluid.layers.auc(input=predict,
+                                                          label=label)
+    cost = fluid.layers.cross_entropy(input=predict, label=label)
+    avg_cost = fluid.layers.mean(x=cost)
+
+    return datas, avg_cost, predict, train_file_path
+
+
+def train(args):
+    datas, avg_cost, predict, train_file_path = model()
+
+    endpoints = args.endpoints.split(",")
+    if args.role.upper() == "PSERVER":
+        current_id = endpoints.index(args.current_endpoint)
+    else:
+        current_id = 0
+    role = role_maker.UserDefinedRoleMaker(
+        current_id=current_id,
+        role=role_maker.Role.WORKER
+        if args.role.upper() == "TRAINER" else role_maker.Role.SERVER,
+        worker_num=args.trainers,
+        server_endpoints=endpoints)
+
+    exe = fluid.Executor(fluid.CPUPlace())
+    fleet.init(role)
+
+    strategy = DistributeTranspilerConfig()
+    strategy.sync_mode = False
```

Comment:
can you give this strategy an algorithmic name rather than sync_mode=False?, I think there can be a lot of methods for async_mode execution like EASGD, DOWNPOURSGD and so on.